### ISSUE TITLE
Cannot connect with Nexus 5X

### ISSUE THREAD

**Issue by mw007** [ubergeek42/weechat-android#277]
I'm trying to use a plain connection (not an ssl tunnel) on my local network. Here are the versions of everything I'm using:

- weechat-android: the stable version (0.11) and the nightly build (0.12-dev).
- weechat 1.2
- android 6.0.1

The behavior I'm seeing is the relay client will connect and send 59 bytes, then will close the connection 20-30 seconds later.

I've tried changing between plain connection and web socket, with no luck.

**Comment by ubergeek42** [comment]
Is your relay in weechat the correct type(weechat, rather than irc). Can you paste the output of `/relay listrelay`

**Comment by mw007** [comment]
It is now working with the stable build. For posterity, I'll post the issue here (or at least what I think the real issue was).

I was indeed using the weechat relay, as opposed to irc. However, when @ubergeek42 mentioned to list the relays, I noticed my relay said it was only listening to IPv6.

```
Listening on ports:
	port 12345, relay: weechat, IPv6, started on: Fri, 18 Mar 2016 09:52:24
```

So I deleted and added the relay again. This time, it's using both IPv4 and IPv6.

```
Listening on ports:
	port 12345, relay: weechat, IPv4+6, started on: Fri, 18 Mar 2016 09:55:01
```

This is very strange, as I obviously did something wrong the first time. But looking through the command history I never specified IPv4 versus IPv6. I simply did `/relay add weechat 12345`.

Chalk it up to me being new to relays I suppose? I'll close the issue as it is indeed working now. Thank you for such a quick reply!
